### DISCUSSION TITLE
What are the long-term goals?

### DISCUSSION THREAD

**Discussion by svenluijten** [Nova-Edge/laravel-openapi#23] [Q&A] [answered]
Hi,

First of all, thank you for forking and offering to maintain the package now that the original author appears to be inactive. I was wondering what the long-term goals for this package are? Will you just be _maintaining_ what already exists, or also adding new features? 

So in short: what can we expect from this fork?

Thanks!

**Comment by TartanLeGrand** [comment, marked as answer]
Hello, sorry for taking so long to reply. Lots of other things to do. The aim is to improve and continue to maintain this lib, which makes it possible to maintain documentation in many projects. 

It should soon be transferred to a workgroup.
